docs/en/docs/advanced/variables.md

```diff
@@ -76,6 +76,7 @@ Variables that support both session-level and global-level setting include:
 * `allow_partition_column_nullable`
 * `insert_visible_timeout_ms`
 * `enable_fold_constant_by_be`
+* `insert_timeout`
```

Comment:
line 361 also need update. You can search with keyword "insert" and "query_timeout" in docs folder and check
description consistency.

Reply:
I did a  global search and updated all the docs I thought demanded.